compiler/damlc/daml-compiler/src/DA/Daml/Compiler/Dar.hs

```diff
@@ -298,7 +298,7 @@ createArchive ::
        PackageConfigFields
     -> String
     -> BSL.ByteString -- ^ DALF
-    -> [(T.Text, BS.ByteString)] -- ^ DALF dependencies
+    -> [(T.Text, BS.ByteString, LF.PackageId)] -- ^ DALF dependencies
```

Comment:
This seems to be completely unrelated to this PR.

Reply:
uh, thanks for spotting this. Seems this slipped in when I pushed after scalafmt.

Reply:
ironically scalafmt still seems unhappy :slightly_smiling_face:

Reply:
scalafmt is trolling me ...